content/en/account_management/org_settings.md

```diff
@@ -136,6 +136,24 @@ Users with the `Org Management` permission can customize the time zone used in a
 
 The setting applies to **all** Monitor alert notifications, as it's an org-wide setting.
 
+#### Max session duration configuration
+
+Users with the `Org Management` permission can set a maximum session duration for their organization. All **new sessions created after this update** follow the configured duration, which applies to **all users**, regardless of their role in the organization. 
```

Comment:
```suggestion
Users with the `Org Management` permission can set a maximum session duration for their organization. The duration applies to all new web sessions created after you change it, for all users, regardless of their role in the organization. It doesn't apply to Datadog mobile application sessions.
```

Reply:
Experimented a bit with combining applicability information in one place so users don't miss the additional info about mobile sessions, but it's not a show-stopper 🙂